exams/pearson/writers_test.py

```diff
@@ -195,13 +195,60 @@ def setUp(self):  # pylint: disable=missing-docstring
         self.cdd_writer = CDDWriter()
         super().setUp()
 
+    def test_first_name(self):  # pylint: disable=missing-docstring,no-self-use
```

Comment:
I'm not convinced that we _need_ to put docstrings on every test -- if a test is short, and the function name is descriptive, that's enough. I've made #2466 to remove the pylint requirement, and I'm happy to argue the point there, if you want. :)

I certainly _could_ use `ddt` here, but I feel like that would make it difficult to understand what the tests are actually doing. `ddt` is best used when you're testing a piece of functionality that should have the same output regardless of what sort of input you throw at it. These tests are asserting that the output should be different based on the input.